RELEASING.md

```diff
@@ -16,18 +36,18 @@
    git push origin X.Y.Z
    ```
 5. Wait for the [publishing workflow](https://github.com/adevinta/spark-android/actions/workflows/publish.yml) to build and publish the release.
-6. Update the `version` in [gradle.properties](gradle.properties) to the next `SNAPSHOT` version.
+6. Update the `version` in [gradle.properties](https://github.com/adevinta/spark-android/blob/main/gradle.properties) to the next `SNAPSHOT` version.
 7. Commit and push the changes to a new PR
   ```bash
   git commit -am "chore: prepare next development version"
   ```
 8. Go to [Sonatype Nexus](https://s01.oss.sonatype.org/) to promote ([docs & detailed steps](https://central.sonatype.org/publish/release/))
-    ##### If there is a problem! and you wanna dismiss it just hit `Drop` 
-    #### If all looks good:
+   ##### If there is a problem! and you wanna dismiss it just hit `Drop`
+   #### If all looks good:
     1. `close` then wait till it completes
-    2. `release` the artifact  
+    2. `release` the artifact
 9. Trigger the manual workflow [![📋 Publish Dokka to GitHub Pages](https://github.com/adevinta/spark-android/actions/workflows/dokka.yml/badge.svg)](https://github.com/adevinta/spark-android/actions/workflows/dokka.yml) with the version tag.
-10. Draft a [new release](https://github.com/adevinta/spark-android/releases/new) with the version tag, add the corresponding [CHANGELOG.md](CHANGELOG.md) entries, and publish it when ready.
+10. Draft a [new release](https://github.com/adevinta/spark-android/releases/new) with the version tag, add the corresponding [CHANGELOG.md](https://github.com/adevinta/spark-android/blob/main/CHANGELOG.md) entries, and publish it when ready.
```

Comment:
same

Reply:
Done in https://github.com/adevinta/spark-android/commit/6730295a22b486d089a1d554ba9e1e2add81719b